includes/functions.php

```diff
@@ -1230,4 +1246,17 @@ function function_check($function) {
 
 function get_last_commit() {
     return `git log -n 1|head -n1`;
-}//end get_last_commit 
+}//end get_last_commit
+
+function snmpTransportToAddressFamily($transport) {
```

Comment:
I think that function is only showing as a new one is added below, as it's nothing to do with this PR we should leave that docblock for another time.